### ISSUE TITLE
Keyword argument `lambshift_S` should overwrite `lambshift`

### ISSUE THREAD

**Issue by neversakura** [USCqserver/OpenQuantumTools.jl#83]

**Comment by paniash** [comment]
Hi! Is this issue still open? Can I have a go at it?

**Comment by neversakura** [comment]
@paniash Yes, the issue is still open. It has been addressed for `solve_ame`, but not for `solve_ule` and `solve_cgme`.

- [x] AME solver
- [ ] CGME solver
- [ ] ULE solver

Please let me know if you need extra help.
